ambry-tools/src/main/java/com.github.ambry/tools/admin/ServerAdminTool.java

```diff
@@ -97,7 +100,14 @@
    * The different operations supported by the tool.
    */
   private enum Operation {
-    GetBlobProperties, GetUserMetadata, GetBlob, TriggerCompaction, RequestControl, ReplicationControl, CatchupStatus
+    GetBlobProperties,
```

Comment:
is this file formatted?

Reply:
Yes, format is done. When file was being formatted, the enum Structure changed. We can manually change it if necessary.

Reply:
No as long as the formatter did this, it is alright